### DISCUSSION TITLE
create_practice_lobby

### DISCUSSION THREAD

**Discussion by nicklson1** [ValvePython/steam#372] [Q&A]
What options should be passed to this function so that there is 1x1 only mid?

**Comment by Gobot1234** [comment]
There is no function called create_practice_lobby in this library. This would be better asked on the dota2 repo. These are the options you have, I'm not sure if that helps https://github.com/ValvePython/dota2/blob/3ca4c43331d8bb946145ffbf92130f52e8eb024a/protobufs/dota_gcmessages_client_match_management.proto#L112-L155
